zerver/models.py

```diff
@@ -1570,6 +1570,36 @@ class UserActivityInterval(models.Model):
     start = models.DateTimeField('start time', db_index=True)  # type: datetime.datetime
     end = models.DateTimeField('end time', db_index=True)  # type: datetime.datetime
 
+class UserDataExport(models.Model):
+    user_profile = models.ForeignKey(UserProfile, on_delete=CASCADE)  # type: UserProfile
+    start = models.DateTimeField('start')  # type: datetime.datetime
+    running = models.BooleanField(default=False, db_index=True)  # type: bool
```

Comment:
We can just remove the row when finished, right?  So we might just need start time.

Actually, we might want to make this table have `(user, start, end, file_path)` -- that would support having a list of past data exports in the UI.